Microsoft.AVS.Management/Microsoft.AVS.Management.psm1

```diff
@@ -229,6 +214,14 @@ function New-AvsLDAPIdentitySource {
         [string]
         $GroupName
     )
+
+    if (-not ($PrimaryUrl -match 'ldap:.*((389)|(636)|(3268)(3269))')) {
```

Comment:
Is the vertical bar missing between (3268) and (3269)

Reply:
good catch! It's been wrong ever since we started